### ISSUE TITLE
Endpoint down

### ISSUE THREAD

**Issue by softliner** [x404xx/Turnstile-Solver#5]
Hey! Sadly, the endpoint is down. Are you looking to get some help maintaining the endpoint?

Softliner

**Comment by x404xx** [comment]
> Hello! Unfortunately, the endpoint is down. Are you seeking for help maintaining it?
> 
> 
> 
> Softliner

Yes, the endpoint is offline. Cloudflare has modified it, and it now only works for invisible captcha.
